pr_agent/cli.py

```diff
@@ -12,7 +13,8 @@ def run():
     parser = argparse.ArgumentParser(description='AI based pull request analyzer')
     parser.add_argument('--pr_url', type=str, help='The URL of the PR to review', required=True)
     parser.add_argument('--question', type=str, help='Optional question to ask', required=False)
-    parser.add_argument('--pr_description', action='store_true', help='Optional question to ask', required=False)
+    parser.add_argument('--pr_description', action='store_true', required=False)
+    parser.add_argument('--pr_code_suggestions', action='store_true', required=False)
```

Comment:
**Suggestion:** Consider using subcommands instead of flags for the different modes of operation (pr_description, pr_code_suggestions). This will make the command line interface more intuitive and easier to use.
```suggestion
    subparsers = parser.add_subparsers(dest='command')

    pr_description_parser = subparsers.add_parser('pr_description')
    pr_code_suggestions_parser = subparsers.add_parser('pr_code_suggestions')
```